### DISCUSSION TITLE
unable to download result

### DISCUSSION THREAD

**Discussion by Zhaowei9908** [MadAnalysis/madanalysis5#34] [Q&A]
Hi all
I was launching the process gg>tt~ and encountered the error
ERROR: MadAnalysis5 failed to create PDF output

**Comment by jackaraz** [comment]
Hi @Zhaowei9908 , have you installed all the [requirements](https://github.com/MadAnalysis/madanalysis5#requirements) for MadAnalysis?

**Reply by Zhaowei9908** [reply]
No,I could't install pdflatex in MA5.I wonder how can I do that

**Reply by jackaraz** [reply]
MadAnalysis does not provide an installation tool for `pdflatex`, you will need to install that externally.

**Reply by Zhaowei9908** [reply]
> Hi @Zhaowei9908 , have you installed all the [requirements](https://github.com/MadAnalysis/madanalysis5#requirements) for MadAnalysis?

The error still exists after I installed all requirments.Besides,there is another error reported by MA5
MA5-ERROR: command root-config is not found

**Reply by jackaraz** [reply]
MadAnalysis can use 3 different plot renderers, gnuplot, root and matplotlib. You can use eitherone of them to plot your results if you dont have root its not a problem.

Have you checked if MadAnalysis can see pdflatex libraries?
